_layouts/lesson.html

```diff
@@ -183,25 +183,21 @@ <h4>{{site.data.snippets.difficulty[page.lang]}}</h4> {% include difficulty.html
   {% include support-alert.html %}
   {% endif %}
 
-  {% unless page == translation_source %}
-  <!-- If this lesson is translated, banner pointing to the original language source of this lesson -->
-  <div class="alert alert-warning">
-    {{ site.data.snippets.translation-source[page.lang] }} {{ site.data.snippets.language-name.en[page.lang]}}: <a
-      href="{{ translation_source.url }}">{{ translation_source.title }}</a>
-  </div>
-  {% endunless %}
 
-  {% for candidate in translation_candidates %}
-  {% unless page.lang == candidate.lang %}
-  <!-- Banner pointing to translations of this lesson when they exist -->
-  <div class="alert alert-warning">
-    {{ site.data.snippets.translation-target[page.lang]}}
-    {{ site.data.snippets.language-name[candidate.lang][page.lang]}}: <a
-      href="{{ candidate.url }}">{{ candidate.title }}</a>
+ {% for candidate in translation_candidates %} 
+<div class="alert alert-warning">
+ 
+  <!-- Banner pointing to the original and other translations of this lesson when they exist -->
+{{ site.data.snippets.available-lesson[page.lang] }} :  <a
+      href="{{ translation_source.url }}"> {{ site.data.snippets.language-name.en[page.lang]}} </a> (original)  |  <a
```

Comment:
This wasn't actually rendering a clickable link, so I took a look at it. Also I think we probably want to specify the source of the translation dynamically rather than assuming it will be English every time, right? I think it should be this instead, but check it out to see if it makes sense to you - 

      href="{{ translation_source.url }}"> {{ site.data.snippets.language-name[translation_source.lang]}} </a>